torch/onnx/symbolic_opset9.py

```diff
@@ -688,6 +688,39 @@ def atan(g: jit_utils.GraphContext, self):
     return g.op("Atan", self)
 
 
+@_onnx_symbolic("aten::atan2")
+# atan2(A, B) = [
+#                   atan(A, B)              if B > 0,
+#                   atan(A, B) + pi         if B < 0, A >= 0,
+#                   atan(A, B) - pi         if B < 0, A < 0,
+#                   pi/2                    if B = 0, A > 0,
```

Comment:
Are these two cases included in above?

Reply:
Yes, last two tests should cover this case